revive.toml

```diff
@@ -19,6 +19,8 @@ warningCode = 1
 [rule.error-strings]
 [rule.errorf]
 [rule.exported]
+[rule.filename-format]
+    arguments=["^[_a-z][_a-z0-9]*.go$"]
```

Comment:
Do you have a need to overload the default one ?

Reply:
Yep, because the default regexp `^[_A-Za-z0-9][_A-Za-z0-9-]*.go$` allows `-` and upper case letters.

Reply:
Indeed, then maybe a comment could be added here.

Reply:
Added